api/coverage.py

```diff
@@ -55,25 +71,10 @@ def __init__(self, collection, lookup_client, **kwargs):
         super(OPDSImportCoverageProvider, self).__init__(collection, **kwargs)
         self.lookup_client = lookup_client
 
-    @classmethod
-    def all(cls, _db, **kwargs):
```

Comment:
Why did you remove this?

Reply:
That was only used by `ContentServerBibliographicCoverageProvider`, which is also removed in this branch. The idea was to make sure that we only used the `lookup` protocol with the content server for collections that ultimately derived from the content server. But we weren't using that -- we get our canonical metadata direct from the metadata wrangler.